src/main/resources/sql/script_29.03.2024.16_07.sql

```diff
@@ -68,11 +68,13 @@ CREATE TABLE `user_roles`
 
 CREATE TABLE `cart`
 (
-    `id`      VARCHAR(36) PRIMARY KEY,
```

Comment:
You need to create migration file for all migrations(for now we have a deployed app and we need to migrations, new sql files invalid for us)